// untitledui-js-base/dist/template.js
import { jsx as r } from "react/jsx-runtime";
import { forwardRef as o } from "react";
const x = o(
  ({
    size: t,
    height: e,
    width: i,
    fill: n,
    strokeWidth: h,
    stroke: w,
    className: s,
    children: l,
    viewBox: a,
    ...d
  }, g) => /* @__PURE__ */ r(
    "svg",
    {
      className: s,
      width: i || t || "24",
      height: e || t || "24",
      viewBox: a || "0 0 24 24",
      fill: n || "none",
      stroke: w || "currentColor",
      strokeWidth: h || "2",
      xmlns: "http://www.w3.org/2000/svg",
      ref: g,
      ...d,
      children: l
    }
  )
), f = o(
  (t, e) => /* @__PURE__ */ r(
    "path",
    {
      ...t,
      stroke: t.stroke || "inherit",
      strokeWidth: t.width || "inherit",
      strokeLinecap: t.strokeLinecap || "round",
      strokeLinejoin: t.strokeLinejoin || "round",
      ref: e
    }
  )
), m = () => /* @__PURE__ */ r(
  "svg",
  {
    width: "24",
    height: "24",
    viewBox: "0 0 24 24",
    xmlns: "http://www.w3.org/2000/svg",
    children: /* @__PURE__ */ r("rect", { x: "2", y: "2", width: "20", height: "20", rx: "4", ry: "4", fill: "gray" })
  }
);
export {
  m as DefaultLazyImportFallback,
  f as Path,
  x as SVG
};
